canister/candid.did

```diff
@@ -35,6 +35,7 @@ type config = record {
   api_access : flag;
   disable_api_if_not_fully_synced : flag;
   watchdog_canister : opt principal;
+  burn_cycles : flag;
```

Comment:
Is this safe? If we don't make it optional would a canister upgrade work fine? I suppose it would because this flag is only used in a request when setting the config (below that request has the flag as an optional field) and the only other place we persist it is in the canister's state which should work given the `#[serde(default)]` has been added?

Reply:
Yes, exactly. We've done this before with other configs.